src/web/modules/gateway_accounts/revoked_api_keys.njk

```diff
@@ -0,0 +1,43 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "common/json.njk" import json %}
+{% extends "layout/layout.njk" %}
+
+{% set isTestData = not (account.type === "live") %}
+
+{% block main %}
+  <h1 class="govuk-heading-m">Revoked API keys for Gateway account {{ gatewayAccountId }}</h1>
+
+  {% for message in messages %}
+    <div class="govuk-error-summary success-summary" role="alert">
+      <h2 class="govuk-error-summary__title">{{message}}</h2>
+    </div>
+  {% endfor %}
```

Comment:
Would we ever have error message(s) in the revoked api keys page?

Reply:
I don't actually know, I've not worked with toolbox enough

Reply:
It looks like the backing controller doesn't do any sort of validation to produce error messages that need to be shown, so the error messages here can be binned.